rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -236,6 +247,31 @@ public String toString() {
                     ", message=" + message +
                     '}';
         }
+
+        @Override
+        public int compareTo(MessageTask messageTask) {
```

Comment:
I think it's weird to have `MessageTask` be naturally ordered by their score. It certainly helps in this implementation, but it is weird to read `taskA > taskB` and assign it any meaning. Maybe use the other constructor to the queue which allows to define a `Comparator<MessageTask>`?

Maybe I'm being too picky here, take just a note/discussion.